apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/utils/columnDefinitions.tsx

```diff
@@ -0,0 +1,420 @@
+          urlPrefix,
+          csrfToken,
+        }),
+      filterFn: numericColumnFilterFn,
+    }),
+
+    // Total points column (hidden by default)
+    columnHelper.accessor('points', {
+      id: 'points',
+      header: 'Total points',
+      cell: (info) =>
+        formatPointsWithEdit({
+          row: info.row.original,
+          field: 'points',
+          hasCourseInstancePermissionEdit: authzDataHasCourseInstancePermissionEdit,
+          urlPrefix,
+          csrfToken,
+        }),
+      filterFn: numericColumnFilterFn,
+      enableHiding: true,
+    }),
+
+    // Score percentage column (not in AI grading mode)
+    ...(!aiGradingMode
+      ? [
+          columnHelper.accessor('score_perc', {
+            id: 'score_perc',
+            header: 'Percentage score',
+            cell: (info) =>
+              formatScoreWithEdit({
+                row: info.row.original,
+                hasCourseInstancePermissionEdit: authzDataHasCourseInstancePermissionEdit,
+                urlPrefix,
+                csrfToken,
+              }),
+            filterFn: numericColumnFilterFn,
+          }),
+        ]
+      : []),
+
+    // Graded by column
+    columnHelper.accessor('last_grader_name', {
+      id: 'last_grader_name',
+      header: 'Graded by',
+      cell: (info) => {
+        const row = info.row.original;
+        if (aiGradingMode) {
+          const showPlus = row.ai_grading_status !== 'None' && row.last_human_grader;
+          return (
+            <span>
+              {row.ai_grading_status !== 'None' && (
+                <span
+                  class={`badge rounded-pill text-bg-light border ${
+                    row.ai_grading_status === 'Graded' || row.ai_grading_status === 'LatestRubric'
+                      ? ''
+                      : 'text-muted'
+                  }`}
+                >
+                  {generateAiGraderName(row.ai_grading_status)}
+                </span>
+              )}
+              {showPlus && ' + '}
+              {row.last_human_grader && <span>{row.last_human_grader}</span>}
+            </span>
+          );
+        } else {
+          if (!info.getValue()) return 'Unassigned';
+          if (row.is_ai_graded) {
+            return (
+              <span class="badge rounded-pill text-bg-light border">{generateAiGraderName()}</span>
+            );
+          }
+          return info.getValue();
+        }
+      },
+      filterFn: (row, columnId, filterValues: string[]) => {
+        if (filterValues.length === 0) return true;
+        const current = row.getValue<InstanceQuestionRow['last_grader_name']>(columnId);
+        if (!current) return filterValues.includes('Unassigned');
+        const rowData = row.original;
+
+        // Check AI grader
+        if (rowData.ai_grading_status !== 'None') {
+          const aiGraderName = generateAiGraderName(rowData.ai_grading_status);
+          if (filterValues.includes(aiGraderName)) return true;
```

Comment:
This could techically be an array-type column, but instead we juts do string searching for parts.